### PR TITLE
add some advice to 'oc debug' for debugging nodes

### PR DESCRIPTION
**Pull request by deads2k**
I had trouble with this command and this help text would have aided me.  See https://github.com/openshift/origin/pull/21997#issuecomment-463208050

@openshift/sig-master 
@sjenning is the phrasing correct?

### CODE REVIEW

File: pkg/oc/cli/debug/debug.go [hunk 1]
```diff
@@ -426,6 +429,9 @@ func (o *DebugOptions) RunDebug() error {
 		} else {
 			fmt.Fprintf(o.ErrOut, "Starting pod/%s ...\n", pod.Name)
 		}
+		if o.IsNode {
+			fmt.Fprintf(o.ErrOut, "To directly access the host PATH, try `chroot /host /bin/bash`\n", pod.Name)
```
Comment (soltysh) [inline pkg/oc/cli/debug/debug.go]: Drop `pod.Name`, `go vet` is complaining about it.

**Comment by smarterclayton** [comment]
Why did you do `chroot /host /bin/bash` instead of `chroot /host`?

**Pull request merged**